meta-resin-common/recipes-support/resin-conf-reset/resin-conf-reset/resin-conf-reset

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -e
```

Comment:
Isn't it much easier to snapshot the empty dir structure at it comes out of yocto and putting it somewhere in `/usr/share` or `/usr/lib`?